packages/packages.json

```diff
@@ -1,41 +1,44 @@
 {
```

Comment:
Please make sure this is backwards compatible. I.e. if I load first time the old format, it should properly drop it like this...

Reply:
Problem with this approach is, the program does not know which packages are local and which packages are third party, so user will have to specify this explicitly.